torchdynamo/output_graph.py

```diff
@@ -433,3 +433,12 @@ def cleanup(self):
         for node in self.graph.nodes:
             if "example_value" in node.meta:
                 del node.meta["example_value"]
+
+    def create_proxy(self, kind, target, args, kwargs, name=None, type_expr=None, proxy_factory_fn=None):
+        rv = super().create_proxy(kind, target, args, kwargs, name, type_expr, proxy_factory_fn)
+
+        # append stack trace to fx node
```

Comment:
Do we need some debug flag to enable this？

Reply:
I'm assuming this track traces will be small, so I don't see a strong need.

Reply:
ok.